data/ecosystem/integrations.yaml

```diff
@@ -181,3 +181,8 @@
   docsUrl: https://farfetch.github.io/kafkaflow/docs/guides/open-telemetry
   components: [.NET]
   oss: true
+- name: Dubbo
+  url: https://github.com/apache/dubbo
+  docsUrl: https://cn.dubbo.apache.org/en/blog/2024/01/31/tracing-dubbo-with-opentelemetry/
```

Comment:
Is there any official documentation for that? A blog post may outdate, so I rather prefer a pointer to the docs that is kept up-to-date accordingly.

Reply:
Hi, I am a member of the Dubbo community. 

It's been a while since we provided support for OpenTelemetry in Dubbo SDK, both apache/dubbo and apache/dubbo-go. But right now, the documentation of this part is only available in Chinese. We are in the process of building an English version of documentation for the whole site, which is expected to be publicly available in the next 2 months.

Can we accept this blog for now? I will update the official documentation when it's available ASAP.

Reply:
Thanks for clarification, @chickenlj !

Can we do the following: Add a comment above that line pointing to this comment (`https://github.com/open-telemetry/opentelemetry.io/pull/3908#discussion_r1473133919`), so we can keep track of it

Beyond that I am then OK with the blog post